### DISCUSSION TITLE
No Such File Or Directory Problem In VS Code

### DISCUSSION THREAD

**Discussion by tbpaksoy** [ocornut/imgui#4697] [New Users Build/Link/Run issues ONLY!]
I have done two things;
1-Put in same file.
2-Added to compilerPath.
But result is like at down
![resim](https://user-images.githubusercontent.com/85836794/140598682-bc86cdd5-332b-4d15-b911-43d7805d3614.png)

How can I solve this ?

**Comment by Delphinidae84** [comment]
It is just what the error message says: There is no such file. A wild guess, maybe the include should look like this:

```cpp
#include "imgui-master/imgui-master/imgui.h"
```

**Reply by PathogenDavid** [reply]
If your project is set up correctly you should not have to do this.

**Comment by rokups** [comment]
It is not clear what build system is used. Anyhow this is a build system question, not Dear ImGui question.
